README.md

```diff
@@ -0,0 +1,66 @@
+# maxwells-daemon
+
+[![Build Status](https://img.shields.io/travis/thumbtack/maxwells-daemon.svg)(https://travis-ci.org/thumbtack/maxwells-daemon)
+[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/aws/aws-sdk-go/blog/master/LICENSE.txt)
```

Comment:
Also, I would link to something more permanent, aws-sdk-go can in theory change its license. Perhaps https://directory.fsf.org/wiki/License:X11 ?